sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -638,7 +638,6 @@ def __init__(
     self.kms_key = kms_key
     self.split_result = None
 
-  @check_accessible(['query'])
```

Comment:
Isn't it correct to ensure that query is accessible, since it may be used in the call to `_start_query_job` below?

Reply:
Ah so you partially answered my question in your comment. Is there any risk of this breaking the `query` path?

Reply:
The query path will still error out if a query is not provided to the template, but it will work fine when it is provided. There are tests for BQ that use the `query` path (see [1],[2])

[1] https://github.com/apache/beam/blob/master/sdks/python/apache_beam/io/gcp/big_query_query_to_table_pipeline.py#L82-L87
[2] https://github.com/apache/beam/blob/master/sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py#L158-L179

Reply:
Gotcha, thanks for the help